packages/brain/src/modules/apiClients/web3signer.ts

```diff
@@ -50,14 +68,41 @@ export class Web3SignerApi extends StandardApi {
         method: "POST",
         endpoint: path.join(this.signEndpoint, pubkey),
         body: JSON.stringify(signerVoluntaryExitRequest),
-        setOrigin: true,
+        headers: this.originHeader,
       });
     } catch (e) {
       e.message += `Error signing (POST) voluntary exit for validator index ${signerVoluntaryExitRequest.voluntary_exit.validator_index}. `;
       throw e;
     }
   }
 
+  /**
+   * Signs a proof of validation for the validator with the specified public key
+   */
+  public async signDappnodeProofOfValidation({
+    signerDappnodeSignRequest,
+    pubkey,
+  }: {
+    signerDappnodeSignRequest: Web3signerPostSignDappnodeRequest;
+    pubkey: string;
+  }): Promise<Web3signerPostSignDappnodeResponse> {
+    try {
+      return await this.request({
```

Comment:
Do you think we need a retry system for these requests?

Reply:
I would be careful when asking web3signer to try again and again for signing keys, might affect attestation performance

Reply:
I would be careful when asking web3signer to try again and again for signing keys, might affect attestation performance